pkg/kubelet/container/helpers.go

```diff
@@ -277,14 +277,6 @@ func SandboxToContainerState(state runtimeapi.PodSandboxState) State {
 	return ContainerStateUnknown
 }
 
-// FormatPod returns a string representing a pod in a human readable format,
-// with pod UID as part of the string.
-func FormatPod(pod *Pod) string {
```

Comment:
Code search confirms this isn't used, wonder why the linter didn't catch this.

https://cs.k8s.io/?q=FormatPod&i=nope&files=&excludeFiles=&repos=kubernetes/kubernetes

Reply:
Yes `FormatPod` isn't used. We checked `.golangci.yaml` but did not find the cause of the undetected problem . We will continue to follow up on this linter problem when have time. Thanks a lot for your review.